packages/backend-plugin-api/src/alpha.ts

```diff
@@ -34,3 +34,19 @@ export const featureDiscoveryServiceRef =
     id: 'core.featureDiscovery',
     scope: 'root',
   });
+
+/**
+ * EXPERIMENTAL: Instance metadata service.
+ *
+ * @alpha
+ */
+export const EXPERIMENTAL_instanceMetadataServiceRef = createServiceRef<
```

Comment:
No need to also use the `EXERIMENTAL_` prefix here, it's already in alpha

Reply:
updated in [c1a3bdb](https://github.com/backstage/backstage/pull/27921/commits/c1a3bdbe1276f7a0a88861f41eff00662b2d9bc5)